APE14.rst

```diff
@@ -233,7 +238,19 @@ low-level API recommends:
             n_pixel scalars or arrays as input, and pixel coordinates should be
             zero-based. Returns n_world scalars or arrays in units given by
             ``world_axis_units``. Note that pixel coordinates are assumed
-            to be 0 at the center of the first pixel in each dimension.
+            to be 0 at the center of the first pixel in each dimension. If a
+            pixel is in a region where the WCS is not defined, NaN can be
+            returned. The coordinates should be specified in the ``(x, y)``
+            order, where for an image, ``x`` is the horizontal coordinate and
+            ``y`` is the vertical coordinate.
+            """
+
+        def numpy_index_to_world_values(self, *index_arrays):
```

Comment:
I am still unsure on putting `numpy` in here, would just 'array' not be more appropriate?

Reply:
Well this was in response to @eteq who pointed out that in future another array package might exist that uses the opposite convention...

Reply:
I find that to be **extremely** unlikely given that the Python language spec is C-order. Also other array packages do current exist that are in C-order. I would rather be inclusive of what current exists rather than worry about things that wont exist beyond midday on April 1st.

Reply:
@eteq - would you be happy to just switch to ``array_index``?

Reply:
Yes, I guess so.  I came to this conclusion totally on my own, not realizing that I'd said the other thing.  So sure, lets go with `array_index`.

Reply:
Changed. I've also removed ``pixel_shape`` and changed ``numpy_shape`` to ``array_shape``.